test/interactive_tarantool.lua

```diff
@@ -263,18 +288,29 @@ function M.new(opts)
     local res = setmetatable({
         ph = ph,
         _readahead_buffer = '',
-        _prompt = 'tarantool> ',
+        _prompt = opts.prompt or 'tarantool> ',
```

Comment:
Nit: I would place `local prompt = opts.prompt` next to `local args = opts.args` for consistency and to have a list of all the options at glance on the function beginning.

Reply:
It doesn't seem necessary, but I am not in the mood to argue about it.